mxcube3/app.py

```diff
@@ -137,8 +136,13 @@ def init(app, hwdir):
 
     @staticmethod
     def _import_adapter_cls(adapter_cls_str):
+        # NBNB Cannot be imported at top level since that causes cirtular imports
+        # NBNB It is no good that a geeneric converter live in a file tat imports
```

Comment:
good point, the logic that does the actual import should perhaps be moved to core.utils (or do you have another
idea). We can keep it as it is in this PR and move it a another PR.

Reply:
It looks like I put htis comment in the wrong place and/or the code has changed over the lst two months.
Anyway, _import_adapter_cls does not seem to be used anywhere currently, so it can easily be left where it is until such a time as we can consider removing it )it does look like a bit of a hack?).

My beef was actually with putting str_to_snake (etc.) in utils as opposed to somewhere in mxcubecore. Which I still think should happen, but which can wait till 1) I need it from somewhere else, or 2) we feel like doing a cleaning day.